src/app/app.component.html

```diff
@@ -1,10 +1,21 @@
+<div
+  *ngIf="
+    !(isLoggedIn$ | async) &&
+    ((dismissedLatestTOS$ | async) !== currentTOSVersion || (dismissedLatestPrivacyPolicy$ | async) !== currentPrivacyPolicyVersion)
+  "
+  class="push-tos"
+>
+  <app-tos-banner class="sticky"></app-tos-banner>
+</div>
 <div class="wrapper">
   <app-banner></app-banner>
   <app-navbar></app-navbar>
-  <router-outlet></router-outlet>
+  <div class="content">
+    <router-outlet></router-outlet>
+  </div>
   <div class="push"></div>
 </div>
-<div class="footer separation">
+<div class="footer">
```

Comment:
Did you remove `separation` class on purpose? If you did, maybe a comment in your final commit message as to why

Reply:
oh I remember now. I originally tried putting the banner at the bottom, but failed. Forgot to put it back. Will do